### ISSUE TITLE
[5.0.0] Start point visualization missing

### ISSUE THREAD

**Issue by printingotb** [Ultimaker/Cura#12681]
### Application Version

5.0.0

### Platform

win10

### Printer

custom fff

### Reproduction steps

See attached project.
Enable visibility for start points in the preview.

### Actual results

Start point representation not visible on all layers/start points.

### Expected results

should be visible

### Checklist of files to include

- [ ] Log file
- [X] Project file

### Additional information & file uploads

![Start points missing](https://user-images.githubusercontent.com/32172100/177575439-7cface66-2df0-4555-b861-9817d725d1a5.jpg)

[Seam test.3mf.zip](https://github.com/Ultimaker/Cura/files/9055682/Seam.test.3mf.zip)

**Comment by Joeydelarago** [comment]
Hello, I've check out the model and I see this issue as well. The team will discuss this.

**Comment by Ghostkeeper** [comment]
Seems to be a visualisation problem. The starts thing is generated in the geometry shader by looking at whether there is a travel move before the extruded move. I think it may have missed the travel move to move the Z axis because that one is not visualised in layer view.

**Comment by Ghostkeeper** [comment]
Considering this issue is pretty harmless, I think we'll have to defer it.

**Comment by BasF0** [comment]
> Seems to be a visualisation problem. The starts thing is generated in the geometry shader by looking at whether there is a travel move before the extruded move. I think it may have missed the travel move to move the Z axis because that one is not visualised in layer view.

I agree, I guess with the arachne engine it is now more likely that the new layer XY coordinates start exactly where the previous layer ended. The visualization can be resolved by faking that the last line of the previous layer was not an extrusion move but a travel move instead. 

I'd fix it but I'm having some trouble calculating the exact indices into the prev_line_types array to replace the type.

**Comment by GregValiant** [comment]
5.13.0 is the latest version.  I'll go ahead and close this.  If it is still an issue a new report would be in order.
